README.md

```diff
@@ -7,6 +7,22 @@ undergraduate CS students.
 
 ## Quick start
 
+### Dependencies
+Before running the scripts, make sure you have the necessary dependencies installed.
+- GNU Build System (Autotools)
+- libncurses
```

Comment:
I think the libncurses library and the support for POSIX thread should be mentioned separate from the other dependencies as there should be some indication that those are always needed to run the `configure` script, while all the other dependencies are only required if the user wants to bootstrap the script themselves with the `bootstrap.sh` script.

Reply:
Makes sense. I will review this

Reply:
Just pushed a new commit. Is this what you requested?